lib/mayaUsd/commands/PullPushCommands.cpp

```diff
@@ -277,7 +280,14 @@ MStatus MergeToUsdCommand::doIt(const MArgList& argList)
         OpUndoItemRecorder undoRecorder(fUndoItemList);
 
         auto& manager = PXR_NS::PrimUpdaterManager::getInstance();
-        status = manager.mergeToUsd(fDagNode, fPulledPath, userArgs) ? MS::kSuccess : MS::kFailure;
+        status = manager.mergeToUsd(dagNode, pulledPath, userArgs) ? MS::kSuccess : MS::kFailure;
+
+        if (status == MS::kSuccess) {
+            // Select the merged prim.  See DuplicateCommand::doIt() comments.
+            Ufe::Selection sn;
+            sn.append(Ufe::Hierarchy::createItem(pulledPath));
+            UfeSelectionUndoItem::select("mergeToUsd", sn);
```

Comment:
Minor: the text is used to help understand the action, so it would be nicer if it said "mergeToUsd: select merged prim".